e2e-tests/tests/features/dbbackup_pg/20-dbb.yaml

```diff
@@ -29,3 +29,55 @@ spec:
   backupStorageName: test-storage
   dbClusterName: test-pg-cluster
 
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: test-pg-cluster-backup-azure
+type: Opaque
+stringData:
+  AZURE_STORAGE_ACCOUNT_NAME: account_name
+  AZURE_STORAGE_ACCOUNT_KEY: account_key
+---
+apiVersion: everest.percona.com/v1alpha1
+kind: BackupStorage
+metadata:
+  name: test-storage-azure
+spec:
+  type: azure
+  bucket: test-backup-bucket
+  credentialsSecretName: test-pg-cluster-backup-azure
+---
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseClusterBackup
+metadata:
+  name: test-db-backup-azure
+spec:
+  backupStorageName: test-storage-azure
+  dbClusterName: test-pg-cluster
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: test-pg-cluster-backup-gcs
+type: Opaque
+stringData:
+  GCS_KEY: gcs_key
+---
+apiVersion: everest.percona.com/v1alpha1
+kind: BackupStorage
+metadata:
+  name: test-storage-gcs
+spec:
+  type: gcs
```

Comment:
IMO we should split these tests to independent ones

Reply:
Running the backups against a single cluster at the same time allows us complete the tests faster and discover concurrency issues, if there are any.